hue/README.md

```diff
@@ -37,3 +37,118 @@ installed:
+        GROUP BY TransactionType;"
+    ```
+   ![Another query](https://github.com/e55010104110/initialization-actions/blob/master/hue/another-query.png)
+   
+****************************************************************************
+
+## Common issues
+
+1.  ### Missing configurations for the distributed Hadoop services
+
+![Hue UI](https://github.com/e55010104110/initialization-actions/blob/master/hue/hue-ui.png)
+
+The hue.ini configuration file is configured to assume a single-node cluster with hostnames set to localhost and port numbers set to default values. For a distributed system, the different sections for each Hadoop service must be updated with the correct hostnames and ports for the servers on which the services are running. 
+
+The hue.sh script performs some generic updates to the hostnames, such as replacing all occurrences of localhost with the fully qualified domain name (FQDN), but additional configurations may be required based on the available services, as described in the documentation for [How to configure Hue for your Hadoop cluster.](https://gethue.com/how-to-configure-hue-in-your-hadoop-cluster/)
+
+2.  ### Changes made to hue.ini are not effective after restart using systemctl {#changes-made-to-hue.ini-are-not-effective-after-restart-using-systemctl}
+
+    `systemctl restart hue.service` leaves behind some orphaned processes and the following command can be used to perform a clean restart of Hue: `sudo /etc/init.d/hue force-stop && sudo /etc/init.d/hue start`
+
+3.  ### User \[hue\] not defined as proxyuser" when integrated with Oozie
+
+    This is caused by missing ProxyUser/impersonation configuration for the logged in user in Oozie.   
+    [Hue.sh](https://github.com/GoogleCloudDataproc/initialization-actions/blob/12792d9d40821e1fad202756e2532a1a8768fe54/hue/hue.sh\#L136) only adds the default user Hue as a ProxyUser, therefore all the other users must also be added. 
+
+1.  Identify the impacted user for example in the error message below it is the  doAs user \`hueuser\`:  *401 Client Error: Unauthorized for url: http://\<redacted\>:11000/oozie/v1/jobs?len=100\&doAs=hueuser\&filter=user%3Dhueadmin%3Bstartcreatedtime%3D-7d\&user.name=hue\&offset=1\&timezone=America%2FLos\_Angeles\&jobtype=wf {"errorMessage":"User \[hue\] not defined as proxyuser","httpStatusCode":401} (error 401\)*  
+
+2.  Add the following properties to oozie-site as follows replacing the \#USER\# with the impacted user in step 1 above. 
+
+    ```
+    <property>
+        <name>oozie.service.ProxyUserService.proxyuser.\#USER\#.hosts\</name>  
+        <value>*</value>  
+    </property>  
+    <property>  
+        <name>oozie.service.ProxyUserService.proxyuser.\#USER\#.groups\</name\> 
+        <value>*</value>  
+    </property>
+    ```
+
+3.  Restart Oozie  
+   1.   Find the process number for the `org.apache.oozie.server.EmbeddedOozieServer  ps \-ef | grep oozie`  
+   2.   Kill the process by PID sudo kill \-9 \<PID\>  
```

Comment:
Stop the Oozie process with sudo systemctl stop oozie.service and if it doesn't stop successfully then you can give this option to kill it with kill -9